Podfile

```diff
@@ -49,7 +49,7 @@ def wordpress_ui
 end
 
 def wordpress_kit
-  pod 'WordPressKit', '~> 14.1'
```

Comment:
I'd be nice to not release a new "major" release of `WordPressKit` every week because it makes it hard to test local changes:

```
[!] CocoaPods could not find compatible versions for pod "WordPressKit":
  In Podfile:
    WordPressAuthenticator (>= 9.0.2, ~> 9.0) was resolved to 9.0.3, which depends on
      WordPressKit (~> 14.0)
```

WordPressKit version should probably be `0.14.1`.